src/tests/data/vcValidationData.json

```diff
@@ -218,38 +218,108 @@
         "ci": null
     },
     "s0200":{
```

Comment:
Can I ask where we got all these expected scores from? The doc linked in `src/services/GenerateVerifiableCredential.ts` doesn't cover them all

Reply:
From this confluence page - https://govukverify.atlassian.net/wiki/spaces/FTFCRI/pages/3557818744/Yoti+Journey+4+Scenarios